_test/fetchphpunit.php

```diff
@@ -17,12 +17,11 @@
     case '7.1':
     case '7.2':
     case '7.3':
-        $phpunit = 'phpunit-7.phar';
-        break;
     case '7.4':
         $phpunit = 'phpunit-7.phar';
-        // PHP 5 backward compatibility lock to PHPUnit 7 (type hinting)
```

Comment:
The reason that we are keeping this line is that PHP 7.4 is supposed to use PHPUnit 8 in the future.

Reply:
Ah, so you just change the php7.4 line into 
$phpunit = 'phpunit-7.phar';
and you're done. that would work